### PR TITLE
Upgrade Ember-CLI

### PR DESCRIPTION
**Pull request by mixonic**
This puts the codebase on a path toward using modern addon APIs, and eventually toward updating Ember-Data, Ember and getting engines.

Todo:

* [ ] Still a bunch of linting issues largely caused by the jshint change. Seems easy to add the "globals" to each file instead
* [ ] Error loading gridiron `Error while processing route: gridiron-admin.index Assertion Failed: Cannot delegate set('organizations', ) to the 'content' property of object proxy (generated gridiron-admin controller): its 'content' is undefined. Error: Assertion Failed: Cannot delegate set('organizations', ) to the 'content' property of object proxy (generated gridiron-admin controller): its 'content' is undefined.`
* [ ] Confirm CI works

### CODE REVIEW

File: public/crossdomain.xml [hunk 1]
```diff
@@ -2,5 +2,14 @@
...
+
+  <!-- Most restrictive policy: -->
   <site-control permitted-cross-domain-policies="none"/>
+
+  <!-- Least restrictive policy: -->
+  <!--
+  <site-control permitted-cross-domain-policies="all"/>
+  <allow-access-from domain="*" to-ports="*" secure="false"/>
+  <allow-http-request-headers-from domain="*" headers="*" secure="false"/>
+  -->
```
Comment (krallin) [inline public/crossdomain.xml]: Can we not add these back? Rationale in: https://github.com/aptible/dashboard.aptible.com/pull/754

Comment (mixonic) [inline public/crossdomain.xml]: Ha, weird. Yup I'll drop 'em.

Comment (krallin) [inline public/crossdomain.xml]: Thanks! 😄

File: tests/acceptance/apps/sidebar-test.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ test('includes breadcrumb to parent stack', function(assert) {
   signInAndVisit(`/apps/${appId}`);
 
   andThen(function() {
-    let stackHandle = find(`h3:contains(${stackHandle})`);
+    let stackHandle = find(`h3`); // TODO this should read :contains(somehandle)
```
Comment (bantic) [inline tests/acceptance/apps/sidebar-test.js]: This should be ok after a merge. It was addressed in #772. In addition to the JSHint error the assertion below is a false-positive (it is essentially: `assert.ok([], 'has stack link')`).

**Pull request closed without merging**